src/tools/gophish_import.py

```diff
@@ -164,21 +164,22 @@ def build_campaigns(api, assessment):
             except Error as e:
                 if e.message == "Template name already in use":
                     logging.warning(
-                        f"{e}. Finding previously loaded template to delete."
+                        "%s. Finding previously loaded template to delete.", e.message
                     )
                     templates = api.templates.get()
                     logging.debug(
-                        f"Checking {len(templates)} for previously imported template to get ID"
+                        "Checking %i for previously imported template to get ID",
+                        len(templates),
                     )
                     for old_template in templates:
                         if old_template.name == new_template.name:
                             logging.debug(
-                                f"Deleting Template with ID {old_template.id}"
+                                "Deleting Template with ID %i", old_template.id
                             )
                             api.templates.delete(old_template.id)
                             logging.info("Re-Loading new template.")
                 else:
-                    logging.error(f"{e}\n")
+                    logging.error("%s", e.message)
```

Comment:
Why not use [`logging.exception()`](https://docs.python.org/3/library/logging.html#logging.Logger.exception) here instead of recreating the wheel?

Reply:
See 44e24edb9ac96eb42759bdea8a9229563811ca59

Reply:
Also been updated here 3ee423033fefa9873fd13190c0e6edb575cd0080